io.github.jliljebl.Flowblade.yaml

```diff
@@ -1,7 +1,7 @@
 app-id: io.github.jliljebl.Flowblade
 default-branch: stable
 runtime: org.gnome.Platform
-runtime-version: '42'
+runtime-version: '44'
```

Comment:
you need to change the plugin version to 22.08

Reply:
and remove Jack as it is in the SD

Reply:
and the python path changed to 3.10

Reply:
Thanks for the input, I'll get on that now.